docs/source/java/cdata.rst

```diff
@@ -220,4 +221,252 @@ Let's create a Java class to test our bridge:
+        return env;
+    }
+
+    int main() {
+        JNIEnv *env;
+        JavaVM *jvm;
+        env = CreateVM(&jvm);
+        if (env == nullptr) return 1;
+        jclass javaClassToBeCalledByCpp = NULL;
+        javaClassToBeCalledByCpp = env->FindClass("ToBeCalledByCpp");
+        if (javaClassToBeCalledByCpp != NULL) {
+            jmethodID fillVector = NULL;
+            fillVector = env->GetStaticMethodID(javaClassToBeCalledByCpp, "fillVector", "(JJ)V");
+            if (fillVector != NULL) {
+                struct ArrowSchema arrowSchema;
+                struct ArrowArray arrowArray;
+                std::cout << "\n<<<<< C++ to Java for Arrays >>>>>\n" << std::endl;
+                env->CallStaticVoidMethod(javaClassToBeCalledByCpp, fillVector, reinterpret_cast<uintptr_t>(&arrowSchema),
+                                          reinterpret_cast<uintptr_t>(&arrowArray));
+                auto resultImportArray = arrow::ImportArray(&arrowArray, &arrowSchema);
+                std::shared_ptr<arrow::Array> array = resultImportArray.ValueOrDie();
+                std::cout << "[C++] Array: " << array->ToString() << std::endl;
+            } else {
+                std::cout << "Problem to read method fillVector\n" << std::endl;
+                return EXIT_FAILURE;
+            }
+            jmethodID fillVectorSchemaRoot = NULL;
+            fillVectorSchemaRoot = env->GetStaticMethodID(javaClassToBeCalledByCpp, "fillVectorSchemaRoot", "(JJ)V");
+            if (fillVectorSchemaRoot != NULL) {
+                struct ArrowSchema arrowSchema;
+                struct ArrowArray arrowArray;
+                std::cout << "\n<<<<< C++ to Java for RecordBatch >>>>>\n" << std::endl;
+                env->CallStaticVoidMethod(javaClassToBeCalledByCpp, fillVectorSchemaRoot,
+                                          static_cast<long>(reinterpret_cast<uintptr_t>(&arrowSchema)),
+                                          static_cast<long>(reinterpret_cast<uintptr_t>(&arrowArray)));
+                auto resultImportVectorSchemaRoot = arrow::ImportRecordBatch(&arrowArray, &arrowSchema);
+                std::shared_ptr<arrow::RecordBatch> recordBatch = resultImportVectorSchemaRoot.ValueOrDie();
+                std::cout << "[C++] RecordBatch: " << recordBatch->ToString() << std::endl;
+            } else {
+                std::cout << "Problem to read method fillVectorSchemaRoot\n" << std::endl;
+                return EXIT_FAILURE;
+            }
+        } else {
+            std::cout << "Problem to read class ToBeCalledByCpp\n" << std::endl;
+            return EXIT_FAILURE;
+        }
+        jvm->DestroyJavaVM();
+        return 0;
+    }
+
+CMakeLists.txt definition file:
+
+.. code-block:: xml
+
+    cmake_minimum_required(VERSION 3.19)
+    project(cdatacpptojava)
+    find_package(JNI REQUIRED)
+    find_package(Arrow REQUIRED)
+    message(STATUS "Arrow version: ${ARROW_VERSION}")
+    message(${ARROW_FULL_SO_VERSION})
```

Comment:
This needs to be updated like the line above

Reply:
Message not needed, deleted